src/snowflake/snowpark/mock/plan.py

```diff
@@ -206,8 +206,15 @@ def handle_order_by_clause(
     sort_columns_array = []
     sort_orders_array = []
     null_first_last_array = []
+    added_columns = []
```

Comment:
these come from the dev/local-testing branch.
I think `local/support-variant` needs to rebase accordingly to the dev/local-testing branch